src/Exception/PasswordLegacyHashException.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Bolt\Exception;
+
+/**
+ * Password legacy hash verification exception.
+ */
+class PasswordLegacyHashException extends PasswordHashException
+{
+    /**
+     * Exception thrown when attempting to use PHPass hashes.
+     *
+     * @return PasswordLegacyHashException
+     */
+    public static function legacyPHPass()
```

Comment:
Why not put this in the constructor? I don't see this class being used for anything else.